ui/src/app/(index)/_components/Announcement.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+import { Alert, AlertDescription, AlertTitle } from '@/components/ui/alert';
+import { AlertCircle } from 'lucide-react';
+
+const Announcement = ({
+    announcement
+}: {
+    announcement: string
+}) => {
+
+    return (
+        <Alert className="bg-yellow-100 border border-yellow-200 mb-2">
+            <AlertCircle data-testid="circle-alert-icon" className="h-4 w-4"/>
+            <AlertTitle className="font-semibold">Announcement</AlertTitle>
+            <AlertDescription>
+                {announcement}
+            </AlertDescription>
+        </Alert>
+    );
+};
```

Comment:
Since there is no longer any TypeScript code right before the return part, we can remove the return word and directly return the component like this:
```
const Announcement = ({
    announcement
}: {
    announcement: string
}) => (
      <Alert className="bg-yellow-100 border border-yellow-200 mb-2">
          <AlertCircle data-testid="circle-alert-icon" className="h-4 w-4"/>
          <AlertTitle className="font-semibold">Announcement</AlertTitle>
          <AlertDescription>
              {announcement}
          </AlertDescription>
      </Alert>
);
```
Note the parenthesis.